docs/csharp/language-reference/builtin-types/char.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "char type- C# reference"
+ms.custom: seodec18
```

Comment:
This should be removed. We were tracking the results of one targeted push for SEO. This new article shouldn't be tracked the same.

Reply:
@BillWagner that's clear: I've removed the line.